Vagrantfile

```diff
@@ -5,7 +5,7 @@ Vagrant.configure(2) do |config|
     postgres: [
       {
        tag: "13.2-alpine",
-       map: "5433:5432", # port mapping
+       map: "5432:5432", # port mapping
```

Comment:
I see what you meant when mentioning Docker in the previous commit huh.

Well, I found it interesting you want to use Vagrant from the beggining. 

I'd suggest being cautious with it.

I mean, just make sure your teammates are familiar with it, if not, let's try to keep it as simple as possible.  For some of your teammates is the first time building web-apps or even it could be their first time writing code in ruby.

Actually, I've never use Vagrant yet, and It'd be really great if you can give us a workshop about it! 🔥 

But, my point is. Let's try to focus on Ruby and RoR for this introductory kata 🥋

Reply:
I agree, in fact we added it because we were having problems installing the same postgres version (13.2), as we have different operating systems. Maybe there's a better or easier way to do this, but I only knew this way and it was what I could share to help my team. I don't know much about vagrant and docker really, besides that docker was an "easy" way to use different versions of postgres and vagrant was an "easy" way to interact with docker.

Actually, only two of us use this configuration. Perhaps we can ignore Vagrantfile and config/database.yml.